corehq/apps/users/static/users/js/custom_data_fields.js

```diff
@@ -32,23 +32,32 @@ hqDefine("users/js/custom_data_fields", [
         self.profile_slug = options.profile_slug;
         self.slugs = options.slugs;
         self.can_edit_original_profile = options.can_edit_original_profile;
+        self.initVals = options.initial_values;
```

Comment:
this initVals is specifically needed to populate the profile field and any other use defined fields (all under the Additional Information category) for when you have the web_user_additional_invite_fields FF enabled. I can't set the initials for those fields directly when they're created because they're represented in the form using crispy Fields (I can't remember the exact details but I went through a lot of trial and error trying to wrangle crispy forms into behaving how I want so I know this is needed!!)

Reply:
I agree that a way to get the initial values is needed. What I meant was that the "Edit User page" also populates initial values, and it uses `user_data` to hold that data. Why can't `user_data` be used here too instead of introducing and separately handling the `initVals` variable? 

The handling logic you added for `initVals` seems identical to the handling of `user_data` which also suggests that it's redundant.

![Screenshot 2024-12-19 at 9 07 03 AM](https://github.com/user-attachments/assets/717bee60-821b-416a-9313-0e769f353dc4)

Reply:
ah gotcha yea that makes sense to me - thanks for pointing out the redundancy!!

Reply:
Thanks for changing it!